usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -0,0 +1,301 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connections, transaction as db_transaction
+from django.db.models import Max
+
+from usaspending_api.awards.models import Award, Subaward
+from usaspending_api.references.models import LegalEntity, Agency, Cfda
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
+from usaspending_api.etl.helpers import get_or_create_location
+
+logger = logging.getLogger('console')
+exception_logger = logging.getLogger("exceptions")
+
+QUERY_LIMIT = 10000
+
+
+class Command(BaseCommand):
+    help = "Load new FSRS data from broker."
+
+    @staticmethod
+    def get_award_data(db_cursor, award_type, max_id):
+        """ Gets data for all new awards from broker with ID greater than the ones already stored for the given
+            award type
+        """
+        query_columns = ['internal_id', 'duns', 'parent_duns', 'principle_place_country', 'principle_place_city',
+                         'principle_place_zip', 'principle_place_state', 'principle_place_street',
+                         'principle_place_district']
+
+        # we need different columns depending on if it's a procurement or a grant
+        if award_type == 'procurement':
+            query_columns.extend(['contract_number', 'idv_reference_number', 'contracting_office_aid', 'company_name',
+                                  'company_address_country', 'company_address_city', 'company_address_zip',
+                                  'company_address_state', 'company_address_street', 'company_address_district'])
+        else:
+            # TODO contracting_office_aid equivalent? Do we even need it?
+            query_columns.extend(['fain', 'awardee_name', 'awardee_address_country', 'awardee_address_city',
+                                  'awardee_address_zip', 'awardee_address_state', 'awardee_address_street',
+                                  'awardee_address_district'])
+        query = "SELECT " + ",".join(query_columns) + " FROM fsrs_" + award_type +\
+                " WHERE id > " + str(max_id) + " ORDER BY id"
+
+        db_cursor.execute(query)
+
+        logger.info("Running dictfetchall on db_cursor")
+        return dictfetchall(db_cursor)
+
+    @staticmethod
+    def get_award(row, award_type):
+        if award_type == 'procurement':
+            # we don't need the agency for grants
+            agency = get_valid_awarding_agency(row)
+
+            if not agency:
+                # logger.warning(
```

Comment:
Is this warning no longer necessary? if it's not, can we just remove it as well the one below that's similar?

Reply:
Crud, I knew I'd forgotten to uncomment something. I commented them out because they were causing too many warnings locally and I wanted to test. We want them printing so we know what internal IDs had problems